site/source/docs/building_from_source/verify_emscripten_environment.rst

```diff
@@ -91,6 +91,6 @@ Other common problems to check for are:
 
    - Errors in the paths in :ref:`.emscripten <compiler-configuration-file>`. These are less likely if you update the file using :ref:`emsdk <emsdk>`.
    - Using older versions of Node or JavaScript engines. Use the default versions for the SDK as listed with :ref:`emsdk list <emsdk>`.
-   - Using older versions of LLVM. The correct versions come with the SDK, but if you're building the environment from source see :ref:`LLVM-Backend` for the proper repos for LLVM and Clang.
+   - Using older versions of LLVM. The correct versions come with the SDK, but if you're building the environment from source you should make sure to use the proper version of LLVM (which you can find using the `emscripten-releases DEPS file and history <https://github.com/emscripten-core/emscripten/blob/incoming/docs/process.md#release-processes>`_).
```

Comment:
You could say that other versions might work, but are not guaranteed to (and are not tested by us)